### PR TITLE
Walker407/service-bell

### PR DESCRIPTION
**Pull request by walker407**
This pr adds interactive functionality to the service bell model.

### CODE REVIEW

File: code/entities/service-bell/ServiceBell.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    /// <summary>
+    /// Set up the model when spawned by the server
+    /// Setup model
+    /// </summary>
+    public override void Spawn()
+    {
+        base.Spawn();
+
+        Transmit = TransmitType.Always;
```
Comment (Remscar) [inline code/entities/service-bell/ServiceBell.cs:20]: Why always?

File: code/entities/service-bell/ServiceBell.Use.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿using Sandbox;
+
+namespace Cinema;
+
+public partial class ServiceBell
+{
+    public string UseText { get; set; } = "Ring Service Bell";
```
Comment (Remscar) [inline code/entities/service-bell/ServiceBell.Use.cs:7]: Nit, could just `UseText => "Ring Service Bell";`

**Review by Remscar — APPROVED** [review]

File: code/entities/service-bell/ServiceBell.cs (hunk 1 above)
Comment (walker407) [reply code/entities/service-bell/ServiceBell.cs:20]: Not sure what to set in this case, just went with what I have seen in other code...  Any suggestions?

File: code/entities/service-bell/ServiceBell.Use.cs (hunk 2 above)
Comment (walker407) [reply code/entities/service-bell/ServiceBell.Use.cs]: Changed

File: code/entities/service-bell/ServiceBell.cs (hunk 1 above)
Comment (Remscar) [reply code/entities/service-bell/ServiceBell.cs:20]: You can remove it. Only set to Always if there's a specific need for it.

Comment (walker407) [reply code/entities/service-bell/ServiceBell.cs:20]: Sounds good, I'll remove it

**Pull request merged**